abscissa_generator/template/src/commands.rs.hbs

```diff
@@ -36,16 +39,25 @@ pub enum {{command_type}} {
 
 /// This trait allows you to define how application configuration is loaded.
 impl Configurable<{{~config_type~}}> for {{command_type}} {
-    // Have `config_path` return `Some(path)` in order to trigger the
-    // application configuration being loaded.
+    /// Location of the configuration file
     fn config_path(&self) -> Option<PathBuf> {
-        None
+        // Check if the config file exists, and if it does not, ignore it.
+        // If you'd like for a missing configuration file to be a hard error
+        // instead, always return `Some(CONFIG_FILE)` here.
+        let filename = PathBuf::from(CONFIG_FILE);
```

Comment:
Maybe I'm missing something, but isn't this going to just try to open `myapp.toml`, without respect to where that config file should actually be located (e.g., `/home/user/.config/`, etc)?

Reply:
Yeah, as configured this will only look in CWD, which is agnostic to the questions you're asking, the answers to which will depend if the application in question is intended to be installed systemwide (several of ours are), or if you're using XDG directories as you suggest, or if you don't want to use XDG directories.

Getting a bit more specific is tricky, and in particular crates like [dirs](https://crates.io/crates/dirs) have a large number of transitive dependencies I'd like to avoid including by default (for my apps that do keep configs in the user's home directory, I'd rather just rely on the [home](https://crates.io/crates/home) crate).

This is definitely something that could use a bit more fleshing out though, and ideally unified with [Abscissa's other path-handling functionality](https://docs.rs/abscissa/latest/abscissa/path/index.html)